packages/shortest/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@antiwork/shortest",
-  "version": "0.1.1",
+  "version": "0.1.2",
```

Comment:
Very important - Every change for the package  requires version bump in order to publish to NPM successfully. Otherwise our publish workflow will fail. Example of a failure: 

![image](https://github.com/user-attachments/assets/99124aec-f222-4602-9c4b-27c8bf2fa536)